File: Lecture11/Databases/app/src/main/java/com/codingblocks/databases/models/Todo.java
package com.codingblocks.databases.models;

/**
 * Created by arnav on 9/24/2017.
 */

public class Todo {
    int id;
    String task;
    boolean done;

    public Todo(int id, String task, boolean done) {
        this.id = id;
        this.task = task;
        this.done = done;
    }

    public Todo(String task, boolean done) {
        this.task = task;
        this.done = done;
    }

    public int getId() {
        return id;
    }

    public String getTask() {
        return task;
    }

    public boolean isDone() {
        return done;
    }
}

File: Lecture17/FirebaseDB/app/src/main/java/com/example/arnav/firebasedb/models/Todo.kt
package com.example.arnav.firebasedb.models

/**
 * Created by arnav on 10/22/2017.
 */

class Todo {
    var task: String = ""
    var isDone: Boolean = false
    var uid: String? = null

    constructor() {
    }

    constructor(task: String, done: Boolean) {
        this.task = task
        this.isDone = done
    }
}
